FilePersistence/src/version_control/merge/ChangeGraph.h

```diff
@@ -70,6 +70,7 @@ class FILEPERSISTENCE_API ChangeGraph
 		void applyNonConflictingChanges(GenericTree* currentTree);
 
 	private:
+		friend class ListMergeComponentV2;
```

Comment:
The ChangeGraph should not know about the `ListMergeComponent` and it cannot be its friend.